lib/clients/AliClient.py

```diff
@@ -0,0 +1,66 @@
+import oss2
+from oss2.headers import RequestHeader
+from .BaseClient import BaseClient
+
+class AliClient(BaseClient):
+    def __init__(self, operation_name, configuration, directory_persistent, directory_work_list, poll_delay_time,
+                 poll_maximum_time):
+        super(AliClient, self).__init__(operation_name, configuration, directory_persistent, directory_work_list,
+                                        poll_delay_time, poll_maximum_time)
+        auth = oss2.Auth(configuration['access_key_id'], configuration['secret_access_key'])
+        # +-> Check whether the given container exists
+        self.container = self.get_container(auth, configuration)
+        if not self.container:
+            msg = 'Could not find or access the given container.'
+            self.last_operation(msg, 'failed')
+            raise Exception(msg)
+
+    def get_container(self, auth, configuration):
+        try:
+            container = oss2.Bucket(auth, configuration['endpoint'], self.CONTAINER)
```

Comment:
What is the endpoint here? As per Alicloud recommendation, we should use internal (can be constructed as given in [Endpoints](https://www.alibabacloud.com/help/doc-detail/31834.htm?spm=a2c63.p38356.b99.26.56bb1746ytYMK6) ), e.g. from ali-dev1 oss-eu-central-1-internal.aliyuncs.com . Can you please check the internal endpoint and make sure we use the internal one?

Reply:
The endpoint here needs to be populated from the manifest. That code change has not been picked up yet. For testing purposes, we tested with internal endpoints as mentioned here.
https://www.alibabacloud.com/help/doc-detail/31837.htm
With `https` protocol.